README.md

```diff
@@ -133,12 +133,30 @@ It is possible to use any of the `manylinux` docker images: `manylinux1`, `manyl
 
 You can define rust extension with RustExtension class:
 
-RustExtension(name, path, args=None, features=None,
-rust\_version=None, quiet=False, debug=False)
+```python
```

Comment:
Thanks, this has long needed some love!

I was actually thinking about also trying to set up a readthedocs page for `setuptools-rust` - I might ask you to review it if that's ok?

Reply:
Sure thing!